README.md

```diff
@@ -47,3 +49,15 @@ module.exports = {
   extends: ['eslint-config-digitalbazaar/vue']
```

Comment:
Isn't the intent of this patch to show composition with `eslint-config-digitalbazaar` and the vue or jsdoc rules?  Same below.

Reply:
I'll add an example of composition in a moment, that said in most cases I think the base rules will show up in the root dir and the vue in the components dir. so I also added a link to the cascade config section of the eslint docs.